.github/workflows/build_and_test_midstream_image.yml

```diff
@@ -31,45 +31,10 @@ jobs:
           podman --version
           cd Dockerfiles/midstream/
           podman build -t midstream_image:latest -f Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0
-      - name: "Run example operator-metadata through midstream ci image  --> positive test"
-        continue-on-error: false
-        run: |
-          mkdir ./output_logs
-          podman run -it -v $PWD:/project/operator-test-playbooks -v ./output_logs:/project/output midstream_image:latest
-
-      - name: "Run example operator-metadata through midstream ci image  --> negative parsing test"
-        continue-on-error: true
-        run: |
-          rm -rf ./output_logs
-          mkdir ./output_logs
-          podman run -it -v $PWD:/project/operator-test-playbooks -v ./output_logs:/project/output -e IMAGE_TO_TEST=quay.io/cvpops/test-operator:missing-alm-examples-v1 midstream_image:latest
 
-      - name: "Succeed if the negative test fails"
+      - name: "Run unit tests"
         continue-on-error: false
-        if: ${{ failure() }}
-        run: |
-          exit 0
-
-      - name: "Fail if the previous job skips"
-        if: ${{ cancelled() }}
         run: |
-          exit 0
-
-      - name: "Run example operator-metadata through midstream ci image  --> negative operator-sdk validation test"
-        continue-on-error: true
-        run: |
-          rm -rf ./output_logs
           mkdir ./output_logs
```

Comment:
@jsztuka please remove this line and also  `-v ./output_logs:/project/output` from podman run.
Such part is unused and just caused issues with SELinux due to following AVC:
```
time->Mon May 31 09:47:37 2021
type=AVC msg=audit(1622468857.005:522): avc:  denied  { relabelto } for  pid=23854 comm="python3" name=".ansible_tmppbp1jnxevalidation-version.txt" dev="dm-0" ino=1582971 scontext=system_u:system_r:container_t:s0:c219,c502 tcontext=system_u:object_r:fusefs_t:s0 tclass=file permissive=0
```

Reply:
Removed both.